torch/csrc/jit/mobile/import.cpp

```diff
@@ -150,6 +153,29 @@ void parseMethods(
         expect_field(table, "register_size", BYTECODE_INDEX_REGISTER_SIZE)
             .toInt();
 
+    // If there exist a tuple in consts_list with this format
+    // ('tensor_jit_index', (4,)) fetch the tensor at index 4 from tensor table
```

Comment:
Any reason the index is put in a tuple? Could it be `('tensor_jit_index', 4)`?

Reply:
Good catch. Will update it.

Reply:
Resolve in the latest commit.